### ISSUE TITLE
Can't filter for genres in music library

### ISSUE THREAD

**Issue by zz0rk** [Komet/MediaElch#842]
Scraper successfully tags musical genres for albums. When I start typing to filter for a genre tag a genre option to select is not being presented. The same functionality works flawlessly for my movie library.

**To Reproduce**
In the filter field start typing to filter for i.e. 'progressive rock'.
The option to select 'Genre: progressiv rock' is not being presented.
Filter for 'progressive rock' returns no results

**Expected behavior**
I expect to see an option to select 'Genre: ...'

**MediaElch Version:**
2.6.2 (App Image)

**Operating System:**
Ubuntu 18.4

**Comment by bugwelle** [comment]
That is correct. MediaElch never supported any filters except for the title for "Music":

https://github.com/Komet/MediaElch/blob/05de1c9b0f9d354cefbffee44acfcd9a652659e7/src/globals/Globals.h#L253-L256

I'll mark this as a feature request. :-)

**Comment by zz0rk** [comment]
Perfect! Thanks
